src/main/java/org/broadinstitute/hellbender/tools/IndexFeatureFile.java

```diff
@@ -39,7 +37,7 @@
 @CommandLineProgramProperties(
         summary = "Creates an index for a feature file, e.g. VCF or BED file.",
         oneLineSummary = "Creates an index for a feature file, e.g. VCF or BED file.",
-        programGroup = VCFManipulationProgramGroup.class
+        programGroup = VariantManipulationProgramGroup.class
```

Comment:
This tool is also mis-categorized I think. The files that this tool can index do not necessarily contain variants. It can index any file format that is supported by Tribble, including things like BED files.

Reply:
True, though "Other" is the only logical choice given the ones we have. @sooheelee this needs a category ?

Reply:
Other is the only one that makes sense.